validator/accounts/v2/wallet_create.go

```diff
@@ -60,11 +60,6 @@ func CreateAndSaveWalletCli(cliCtx *cli.Context) (*wallet.Wallet, error) {
 
 // CreateWalletWithKeymanager specified by configuration options.
 func CreateWalletWithKeymanager(ctx context.Context, cfg *CreateWalletConfig) (*wallet.Wallet, error) {
-	if err := wallet.Exists(cfg.WalletCfg.WalletDir); err != nil {
-		if !errors.Is(err, wallet.ErrNoWalletFound) {
-			return nil, errors.Wrap(err, "could not check if wallet exists")
-		}
-	}
```

Comment:
Reason why this was removed:

If err is not nil then it can be that there is nothing at that file path.  This is actually desirable for new wallet creation. Otherwise other errors would only occur in the process of this performing this check.  

The previous Exists() also superficially checked for wallet directory validity:  A nil error result is actually not desirable as that means a [valid] wallet already exists at this location.  But a check for wallet already-present occurs in CreateAndSaveWalletCli and the other callers of this function are tests which depend on creating the folder prior to doing their other activities.  There is an wallet creation API handler that calls this function but does its own check.  As there is a test dedicated to the circumstance where a wallet already exists and ensuring correct behavior, I did not add (or move) the check for an already-present wallet here.

Reply:
> As there is a test dedicated to the circumstance where a wallet already exists and ensuring correct behavior, I did not add (or move) the check for an already-present wallet here.

Which test is it?

Reply:
TestCreateWallet_WalletAlreadyExists() in validator/accounts/v2/wallet_create_test.go